universe/NamedValueRefManager.h

```diff
@@ -272,12 +272,14 @@ class FO_COMMON_API NamedValueRefManager {
     V* GetValueRefImpl(const std::map<NamedValueRefManager::key_type, std::unique_ptr<V>, std::less<>>& registry,
                        std::string_view label, std::string_view name) const
     {
-        TraceLogger() << "NamedValueRefManager::GetValueRef look for registered " << label << " valueref for \"" << name << '"';
-        TraceLogger() << "Number of registered " << label << " ValueRefs: " << registry.size();
+        TraceLogger() << "NamedValueRefManager::GetValueRef look for registered (" << label << ") valueref for \"" << name << '"';
+        TraceLogger() << "Number of registered (" << label << ") ValueRefs: " << registry.size();
         const auto it = registry.find(name);
         if (it != registry.end())
             return it->second.get();
-        WarnLogger() << "NamedValueRefManager::GetValueRef found no registered " << label << " valueref for \"" << name << "\". This should not happen once \"#3225 Refactor initialisation of invariants in value refs to happen after parsing\" is implemented";
+        DebugLogger() << "NamedValueRefManager::GetValueRef found no registered (" << label << ") valueref for \"" << name
+                      << "\". This is either looking in the wrong registry (which can be OK). This should not happen "
+                      << "if looking in the right registry once \"#3225 Refactor initialisation of invariants in value refs to happen after parsing\" is implemented";
```

Comment:
```suggestion
        DebugLogger() << "NamedValueRefManager::GetValueRef found no registered (" << label << ") valueref for \"" << name
                      << "\". This is maybe be due to looking in the wrong registry (which can be OK). This should not happen "
                      << "if looking in the right registry once initialisation of invariants in valuerefs to happen after parsing\" is implemented";
```

never liked the reference to a github issue by number in the code... would prefer to remove it editing this anyway...

Reply:
actually i think initialisation of invariants is not related (anymore) to missing named values.

initialisation of invariants does not affect the registration (currently) - so even if it is done suboptimal/undeterministic (or gets changed in a later pass in order to make it optimal/deterministic) its unrelated. Only if we made registration of named values lazy (until the invariant is known) there might be connection. But I do not see that coming, so I will simply remove the reference to invariants.

Reply:
> initialisation of invariants does not affect the registration

I thought the issue was that named refs needed to be resolved to determine the invariants of refs that look them up?